src/test/groovy/wooga/gradle/node/NodeReleasePluginSpec.groovy

```diff
@@ -50,7 +51,7 @@ class NodeReleasePluginSpec extends ProjectSpec {
         project.tasks.getByName(taskName)
 
         where:
-        taskName << [NodeReleasePlugin.NPM_TEST_TASK, NodeReleasePlugin.NPM_BUILD_TASK, NodeReleasePlugin.NPM_CLEAN_TASK]
+        taskName << [NodeReleasePlugin.NODE_TEST_TASK, NodeReleasePlugin.NODE_BUILD_TASK, NodeReleasePlugin.NODE_CLEAN_TASK]
```

Comment:
```suggestion
        taskName                          | _
        NodeReleasePlugin.NODE_TEST_TASK  | _
        NodeReleasePlugin.NODE_BUILD_TASK | _ 
        NodeReleasePlugin.NODE_CLEAN_TASK | _
```

Might make it easier to add/remove tasks in the future.

One aspect I also struggle with this is that you actually test an interface here. That the plugin applies a task with name `X`. That you do. But if you change the value of the constants, your test won't warn you if you broke your API (in this case the tasks the user expects to be added after the apply). Only if your code suddenly fails to add a task will this test fail. I started to use simple strings in this cases and treat the test spec as a Spec.